presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -64,6 +64,10 @@ class TaskManager {
           std::unordered_map<std::string, std::string>>&&
           connectorConfigStrings);
 
+    std::unique_ptr<protocol::TaskInfo> executeResultRequests(const PrestoTaskPtr prestoTask,
```

Comment:
Please, document this method.

Use const & for prestoTask and resultRequests parameters.

A better name might be processResultRequests.